flexget/components/sites/sites/piratebay.py

```diff
@@ -103,7 +103,7 @@ def url_rewritable(self, task, entry):
         url = entry['url']
         if url.endswith('.torrent'):
             return False
-        return bool(self.url_match.match(url))
+        return bool(self.url_match.match(url) or self.url_search.match(url))
```

Comment:
I'm not sure what the logic here is supposed to do

Reply:
The old `url_match` regex used to match both the search url and the regular urls...now it does not.

Reply:
Unclear if that behavior is necessary but we have a test that makes sure it works so i figured we'd expect it to. 
https://github.com/Flexget/Flexget/blob/c7a6ab37c27b11afec6e71503efe2398d8d748cd/flexget/tests/test_urlrewriting.py#L47-L52

Reply:
ok, how about something like this then:
```suggestion
        return any((self.url_match.match(url), self.url_search.match(url)))
```

Reply:
`any` and `or` both feel fine to me. I'd ditch the `bool()` cast if we stick with `or` though.

Reply:
I think the bool cast is still necessary if we use `or` since otherwise it will return the match object. But i can just change it to any if that seems cleaner.